### DISCUSSION TITLE
Question about removing `docker.userEmulation `

### DISCUSSION THREAD

**Discussion by mirpedrol** [nextflow-io/nextflow#4601] [Q&A]
Hello!

The option `docker.userEmulation` has been removed with https://github.com/nextflow-io/nextflow/pull/4596, but in nf-core we found some situations where we used this option to fix permission errors.

For example, in modules where inside containers the user is `biodocker`. This causes an error where `.command.trace` can't be created. See https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/ncbitools/vecscreen/nextflow.config#L11-L16

Is there a different solution for these situations?

**Comment by pditommaso** [comment]
Hi Julia, I guess it should be possible to replace it with `process.containerOptions = '-u $(id -u):$(id -g)'`

**Reply by mirpedrol** [reply]
Perfect, thank you! I will try this
